### PR TITLE
[Needs developer review] Edits to step 3 of remote-deploying-codewind.md

### PR DESCRIPTION
**Pull request by sishida**
Signed-off-by: Sarah Ishida <sishida@us.ibm.com>

Edits from developer feedback.

PR for issue https://github.com/eclipse/codewind/issues/2720.

### CODE REVIEW

File: docs/_documentations/remote-deploying-codewind.md [hunk 1]
```diff
@@ -72,22 +72,28 @@ devex001   codewind    122m   https://codewind-keycloak-devex001.<ingress-domain
...
-1\. Copy the **Access URL** returned in the previous step and paste it into a browser.  
+1. Copy the **Access URL** and paste it into a browser.
 
-2\. Ensure that the Realm is set to `Codewind` by clicking on the dropdown arrow on the page. Select **Codewind** if necessary, then:
+2. Click **Administration Console** and log in to Keycloak with `admin` as the default login user name and `admin` as the default password.
 
+3. Change the administrator password and return to the **Administration Console**.
+- Click the **Admin** link and choose **Manage Account/Password**.
+- Set a new administrator password.
+- To return to the admin console, use the **Admin** link or log out and log back in to Keycloak as the admin user with your new admin password.
```
Comment (eharris369) [inline docs/_documentations/remote-deploying-codewind.md:82]: Actually, the link to return to the admin console is: 
image

**Comment by sishida** [comment]
@eharris369 Thanks for pointing out the correct name of the link! I've added a new commit to fix it. :)

**Comment by sishida** [comment]
@eharris369 Made some tiny ID edits in latest commit. Would it be accurate to say, "The field **Temporary = On** requires you to change your password during first connection." for line 94? Trying to use second person instead of "users" when it makes sense...Thanks!

**Comment by eharris369** [comment]
Yes, that looks good to me.

**Pull request merged**